lib/error.js

```diff
@@ -340,6 +341,11 @@ AppError.tooManyRequests = function (retryAfter, retryAfterLocalized) {
     extraData.retryAfterLocalized = retryAfterLocalized
   }
 
+  if (canUnblock) {
+    extraData.verificationMethod = 'email-captcha'
```

Comment:
nit: are we keeping this "email-captcha" name for historical reasons, or should we call it something else?